native/common/jp_classhints.cpp

```diff
@@ -85,7 +85,10 @@ JPMethodMatch::JPMethodMatch(JPJavaFrame &frame, JPPyObjectVector& args, bool ca
 }
 
 JPConversion::~JPConversion() = default;
-JPClassHints::JPClassHints() = default;
+JPClassHints::JPClassHints()
+{
+	convertJava = false;
```

Comment:
please keep the "m_" prefix for member variable code style. And maybe use initializer lists.